frontend/src/components/material/MaterialTable.vue

```diff
@@ -330,42 +337,32 @@ export default {
           id: key,
           quantity: mi.quantity,
           unit: mi.unit,
+          combinedQuantity: this.renderQuantity(mi),
           article: mi.article,
           listName: this.materialLists.find(
             (listItem) => listItem.value === mi.materialList
           ).text,
           new: true,
           serverError: mi.serverError,
           readonly: true,
-          class: 'new', // CSS class of new item rows
+          rowClass: 'new', // CSS class of new item rows
         })
       }
 
       return items
     },
-    materialItemsSorted() {
-      const items = this.materialItemCollection.items
-
-      // eager add new Items
-      for (const key in this.newMaterialItems) {
-        const mi = this.newMaterialItems[key]
-        items.push({
-          id: key,
-          quantity: mi.quantity,
-          unit: mi.unit,
-          article: mi.article,
-        })
-      }
-
-      return items.sort((a, b) => a.article.localeCompare(b.article))
-    },
   },
   methods: {
-    // remove existing item
-    deleteMaterialItem(materialItem) {
-      this.api
-        .del(materialItem.uri)
-        .catch((e) => this.$toast.error(errorToMultiLineToast(e)))
+    renderQuantity(item) {
```

Comment:
someone said that you should not use methods to map computed things.
maybe this quantity should be in a separate component?
Can also be done in a later PR (but i fear then it will never happen)

Reply:
The methods inside computed should only process/return params. This is not a problem.